packages/manifest/src/index.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TeamsAppManifest, IStaticTab, IConfigurableTab } from "@microsoft/teamsfx-api";
+import * as fs from "fs-extra";
+
+export class Manifest {
+  private manifest: TeamsAppManifest;
+
+  constructor(appName: string) {
+    const manifest =  new TeamsAppManifest();
+    manifest.name.short = appName;
+    manifest.name.full = appName;
+    this.manifest = manifest;
+  }
+  /**
+   * Load manifest from manifest.json
+   * @param filePath path to the manifest.json file
+   * @throws FileNotFoundError - when file not found
+   * @throws InvalidManifestError - when file is not a valid json or can not be parsed to TeamsAppManifest
+   */
+  async load(filePath: string): Promise<TeamsAppManifest> {
+    this.manifest = await fs.readJson(filePath);
+    return this.manifest;
+  }
+
+  /**
+   * Save manifest to .json file
+   * @param filePath path to the manifest.json file
+   */
+  async save(filePath: string): Promise<void> {
+    await fs.writeFile(filePath, JSON.stringify(this.manifest, null, 4));
+  }
+
+  /**
+   * Validate manifest against json schema
+   * @returns {string[]} An array of error string
+   */
+  async validate(): Promise<string[]> {
+    return [];
+  }
+
+  /**
+   * Register a Teams app or update the existing Teams app
+   * @param filePath path to the zip file, with manifest and icons
+   * @param accessToken Access token with https://dev.teams.microsoft.com/AppDefinitions.ReadWrite scope
+   * @returns {string} Teams app id
+   */
+  async deploy(filePath: string, accessToken: string): Promise<string> {
+    return "";
+  }
+
+
+  /// strech goal
+
+  /**
+   * Provide an initial manifest file, without capabilities
+   * @param appName short name for Teams app
+   * @param filePath path to the manifest.json file
+   */
+  async init(appName: string, filePath: string): Promise<void> {}
```

Comment:
Perhaps this should be static and return TeamsAppManifest, given that we already have `save()` method.